packages/core/android/src/oldarch/java/io/sentry/react/RNSentryModule.java

```diff
@@ -178,11 +178,21 @@ public void crashedLastRun(Promise promise) {
     this.impl.crashedLastRun(promise);
   }
 
-  @ReactMethod()
+  @ReactMethod
   public void getNewScreenTimeToDisplay(Promise promise) {
     this.impl.getNewScreenTimeToDisplay(promise);
   }
 
+  @ReactMethod
+  public void getDataFromUri(String uri, Promise promise) {
```

Comment:
`getDataFromUri` is already included [in line 156 above](https://github.com/getsentry/sentry-react-native/pull/4751/files#diff-f80f4dbdb26acf795dd444f5c769c7b27c1b9c74c0a5f0b2e553dbec962edc0cR156). Let's remove method from the old position and keep this one for more consistent order.

Reply:
I've missed that, thanks.